src/main/java/com/softserve/bookstore/service/BookService.java

```diff
@@ -26,9 +26,9 @@ public List<Book> getBooksFromFile(String fileName) throws IOException {
     }
 
 
-    public void findAuthorById(String fileName) throws SQLException, IOException {
-        List<Book> books = getBooksFromFile(fileName);
-        bookRepository.findAll();
+    public List<Book> findAllBooks() throws SQLException {
+        return bookRepository.findAll();
+
```

Comment:
not a big deal, but don't leave to many blank lines, if neccessary one blank line is just fine
but don't use two
in this case I would delete both lines

in most cases blank lines are just used for spacing between different logic